apps/src/levelbuilder/lesson-editor/activitiesEditorRedux.js

```diff
@@ -85,10 +85,15 @@ export const updateActivitySectionField = (
   fieldValue,
 });
 
-export const addActivitySection = (activityPosition, activitySectionKey) => ({
+export const addActivitySection = (
+  activityPosition,
+  activitySectionKey,
+  insertPosition
```

Comment:
You're inheriting some rather confusing code here, so my suggestions are mostly in light of reducing confusion you didn't cause. At a casual glance `activityPosition` and `insertPosition` are redundant. Really, they should have passing an `activity` or `acitivtyId` in here (or whatever), not its `activityPosition` 🙈. I can see why that was convenient when this was written, but its not great.

You could help clarify this argument without changing its logic by maybe changing yours to:
`activitySectionInsertPosition`. Its a mouthful, but I think its justified in that it also clarifies the other argument.

Reply:
Or, alternatively: `activitySectionPosition`, assuming this becomes the position of the new activitySection in the list? (I think it does, but worth double-checking).